script/generic/asset-infos.ts

```diff
@@ -67,7 +68,14 @@ function isAssetInfoValid(info: unknown, path: string, address: string, chain: s
 
     // status
     if (!isValidStatusValue(info['status'])) {
-        return [`Invalid value for status field, '${info['status']}'`, "", fixedInfo]
+        return [`Invalid value for status field, '${info['status']}'`, "", fixedInfo];
+    }
+
+    // tags
+    if (info['tags']) {
+        if (!isValidTagValues(info['tags'])) {
+            return [`Invalid tags, '${info['tags']}'`, "", fixedInfo];
```

Comment:
Can you check that tag contain in tags.json?

Reply:
That's what function isValidTagValues does